.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,32 @@
+# bump-version bumps a new version
+# runs only if it passes all the checks
+name: bump-version
+
+on:
+  workflow_run:
+    workflows: [check-build-test]
+    types: [completed]
+
+jobs:
+  on-success:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
-> if: ${{ github.event.workflow_run.conclusion == 'success' **&& check-test-build is triggered by push to master** }}

Hey @wxharry, since the workflow "check-build-test" will be triggered by both push and PR so we should add another condition to make sure "bump-version" only be triggered after pusshing to master.

see: https://github.com/hypertrons/hypertrons-crx/pull/352#issuecomment-1146628296

Reply:
Since bump-version is triggered by workflow_run, I have no idea how to access the information on how that workflow was triggered.
Can we put bump-version and check-test-build in one single workflow so that we know with condition to bump a version?

Reply:
Hey @wxharry, I think ${{ github.event_name }} would help. We can use it in if-branch.

Please see:

- https://github.com/tyn1998/workflows-for-hypercrx/pull/9/files
- https://github.com/tyn1998/workflows-for-hypercrx/runs/6850861699?check_suite_focus=true
- https://github.com/tyn1998/workflows-for-hypercrx/runs/6850864749?check_suite_focus=true

Reply:
oops, sorry. it is triggered by **workflow_run!**

Reply:
Hey @wxharry, ${{ github.event.workflow_run.event }} is the anwser!

see:

- https://github.com/tyn1998/workflows-for-hypercrx/runs/6850962732?check_suite_focus=true
- https://github.com/tyn1998/workflows-for-hypercrx/runs/6850967950?check_suite_focus=true

| pull_request | push |
| --- | --- |
| <img width="552" alt="image" src="https://user-images.githubusercontent.com/32434520/173242099-94d963cd-fdcb-41a0-984a-8451d9ed7179.png"> | <img width="670" alt="image" src="https://user-images.githubusercontent.com/32434520/173242134-e1e56162-3f7e-4b95-851d-51ad085de894.png"> |

Reply:
Got it